### DISCUSSION TITLE
请问有vue2.0的库吗

### DISCUSSION THREAD

**Discussion by crosscloudmoon** [zouyaoji/vue-cesium#422] [General]
跪求！

**Comment by zouyaoji** [comment]
已经没维护了 不推荐！
https://github.com/zouyaoji/vue-cesium-v2

**Comment by crosscloudmoon** [comment]
谢谢🙏。了解了
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年08月10日 20:48 |
| 收件人 | zouyaoji/vue-cesium ***@***.***> |
| 抄送人 | MagiBox ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [zouyaoji/vue-cesium] 请问有vue2.0的库吗 (Discussion #422) |

已经没维护了 不推荐！
https://github.com/zouyaoji/vue-cesium-v2
